src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
@MatApple Then the Universe option is only available to enterprise customers; this option should be available to all customers.